### PR TITLE
[terra-form-select] - Fixed duplicate theme variable

### LINKED ISSUE #1727: Form-Select: additional themeable variable
# Bug Report

## Description

The themeable variable `--terra-select-padding-bottom` is currently set to have two different default values:

```
.default { 
    ...
    padding-bottom: var(--terra-select-padding-bottom, 0);
```
https://github.com/cerner/terra-core/blob/d2e933158c8a88c1cb92e0bbf1a6cfacd78961b4/packages/terra-form-select/src/_Frame.module.scss#L17

and 
 
```
.tag,
.search,
.combobox,
.multiple {
> .display {
      ...
      padding-bottom: var(--terra-select-padding-bottom, 0.21428571428571427rem);
```
https://github.com/cerner/terra-core/blob/d2e933158c8a88c1cb92e0bbf1a6cfacd78961b4/packages/terra-form-select/src/_Frame.module.scss#L195

## Possible Solution

Add new padding variables (for .tag, .search, .combobox, .multiple) to provide correct theme-ability options. 

Also may need to add new variable for input height:
https://github.com/cerner/terra-core/blob/d2e933158c8a88c1cb92e0bbf1a6cfacd78961b4/packages/terra-form-select/src/_Frame.module.scss#L156

### PR DESCRIPTION
**Pull request by StephenEsser**
### Summary
- Removed duplicate theme value
- Added theme variable for the input height

The Select had a theme variable defined twice with different values making theming difficult. 

This change enables the combobox, multiple, and tag variants to have a common theme-able wrapper and removes the duplicate variable.

### Additional Details
Adds theme-able padding variables to the red outline below.
image

Resolves #1727 

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-form-select/src/_Frame.module.scss [hunk 1]
```diff
@@ -120,14 +120,17 @@
     padding: var(--terra-select-toggle-padding, 0 0.42857rem);
   }
 
-  .tags {
+  .content {
```
Comment (StephenEsser) [inline packages/terra-form-select/src/_Frame.module.scss]: I'm struggling to find a good name for this content region, if anyone has any ideas let me know. This content region is the wrapper around the input and selected values for tag, multiple, and combobox variants. The default variant does not have this wrapper.

Comment (bjankord) [reply packages/terra-form-select/src/_Frame.module.scss]: `.input-container` is what comes to mind. I'm cool with `.content`.

Comment (yuderekyu) [reply packages/terra-form-select/src/_Frame.module.scss]: if you want to be more specific, maybe `non-default-content`?

**Comment by neilpfeiffer** [comment]
Tested: PR #1939
Resolves: Issue #1727

**Reviewed:**
- Looks good, only alternate naming suggestion (could consider for the future) for the Frame-module might be something like:  `.display  -->  . outer` and `.content  -->  .inner`

\
\
Verified by @neilpfeiffer, Ready for Release.

**Pass** |
-----------|
UX Review|

**Pull request merged**